taglib/toolkit/tbytevector.h

```diff
@@ -567,6 +567,16 @@ namespace TagLib {
      */
     ByteVector toHex() const;
 
+    /*!
+     * Returns a base64 encoded copy of the byte vector
+     */
+    ByteVector toBase64() const;
```

Comment:
We should have many explicit tests for the base64 code, both encoding and decoding.